ComAmazonawsKms/runtimes/python/src/com_amazonaws_kms/smithygenerated/com_amazonaws_kms/dafny_to_aws_sdk.py

```diff
@@ -0,0 +1,2194 @@
+    ConnectionStateType_CONNECTING,
+    ConnectionStateType_DISCONNECTED,
+    ConnectionStateType_DISCONNECTING,
+    ConnectionStateType_FAILED,
+    CustomerMasterKeySpec_ECC__NIST__P256,
+    CustomerMasterKeySpec_ECC__NIST__P384,
+    CustomerMasterKeySpec_ECC__NIST__P521,
+    CustomerMasterKeySpec_ECC__SECG__P256K1,
+    CustomerMasterKeySpec_RSA__2048,
+    CustomerMasterKeySpec_RSA__3072,
+    CustomerMasterKeySpec_RSA__4096,
+    CustomerMasterKeySpec_SYMMETRIC__DEFAULT,
+    DataKeyPairSpec_ECC__NIST__P256,
+    DataKeyPairSpec_ECC__NIST__P384,
+    DataKeyPairSpec_ECC__NIST__P521,
+    DataKeyPairSpec_ECC__SECG__P256K1,
+    DataKeyPairSpec_RSA__2048,
+    DataKeyPairSpec_RSA__3072,
+    DataKeyPairSpec_RSA__4096,
+    DataKeySpec_AES__128,
+    DataKeySpec_AES__256,
+    EncryptionAlgorithmSpec_RSAES__OAEP__SHA__1,
+    EncryptionAlgorithmSpec_RSAES__OAEP__SHA__256,
+    EncryptionAlgorithmSpec_SYMMETRIC__DEFAULT,
+    ExpirationModelType_KEY__MATERIAL__DOES__NOT__EXPIRE,
+    ExpirationModelType_KEY__MATERIAL__EXPIRES,
+    GrantOperation_CreateGrant,
+    GrantOperation_Decrypt,
+    GrantOperation_DescribeKey,
+    GrantOperation_Encrypt,
+    GrantOperation_GenerateDataKey,
+    GrantOperation_GenerateDataKeyPair,
+    GrantOperation_GenerateDataKeyPairWithoutPlaintext,
+    GrantOperation_GenerateDataKeyWithoutPlaintext,
+    GrantOperation_GetPublicKey,
+    GrantOperation_ReEncryptFrom,
+    GrantOperation_ReEncryptTo,
+    GrantOperation_RetireGrant,
+    GrantOperation_Sign,
+    GrantOperation_Verify,
+    KeyManagerType_AWS,
+    KeyManagerType_CUSTOMER,
+    KeySpec_ECC__NIST__P256,
+    KeySpec_ECC__NIST__P384,
+    KeySpec_ECC__NIST__P521,
+    KeySpec_ECC__SECG__P256K1,
+    KeySpec_RSA__2048,
+    KeySpec_RSA__3072,
+    KeySpec_RSA__4096,
+    KeySpec_SYMMETRIC__DEFAULT,
+    KeyState_Creating,
+    KeyState_Disabled,
+    KeyState_Enabled,
+    KeyState_PendingDeletion,
+    KeyState_PendingImport,
+    KeyState_PendingReplicaDeletion,
+    KeyState_Unavailable,
+    KeyState_Updating,
+    KeyUsageType_ENCRYPT__DECRYPT,
+    KeyUsageType_SIGN__VERIFY,
+    MessageType_DIGEST,
+    MessageType_RAW,
+    MultiRegionKeyType_PRIMARY,
+    MultiRegionKeyType_REPLICA,
+    OriginType_AWS__CLOUDHSM,
+    OriginType_AWS__KMS,
+    OriginType_EXTERNAL,
+    SigningAlgorithmSpec_ECDSA__SHA__256,
+    SigningAlgorithmSpec_ECDSA__SHA__384,
+    SigningAlgorithmSpec_ECDSA__SHA__512,
+    SigningAlgorithmSpec_RSASSA__PKCS1__V1__5__SHA__256,
+    SigningAlgorithmSpec_RSASSA__PKCS1__V1__5__SHA__384,
+    SigningAlgorithmSpec_RSASSA__PKCS1__V1__5__SHA__512,
+    SigningAlgorithmSpec_RSASSA__PSS__SHA__256,
+    SigningAlgorithmSpec_RSASSA__PSS__SHA__384,
+    SigningAlgorithmSpec_RSASSA__PSS__SHA__512,
+    WrappingKeySpec_RSA__2048,
+)
+
+
+def com_amazonaws_kms_AlreadyExistsException(input):
+    output = {}
+    if input.message.is_Some:
+        output["message"] = input.message.value.VerbatimString(False)
```

Comment:
What does `input.KeyId.VerbatimString(False)` do?

Reply:
This converts a Dafny key ID to a native key ID
Dafny key ID is a `_dafny.Seq`. Seq has a method VerbatimString: [link](https://github.com/dafny-lang/dafny/blob/385103dd9e77a329b04c45fef0c3ceee1ae6bb26/Source/DafnyRuntime/DafnyRuntimePython/_dafny.py#L155)
Dafny strings are represented as tuples of characters in Python.
VerbatimString(False) will `''.join` the Dafny characters together.
This gets you to a native string